swo_aws_extension/aws/client.py

```
@@ -192,10 +276,10 @@ def _get_access_token(self):
 
     @wrap_boto3_error
     def _get_credentials(self):
-        if not self.pma_account_id:
+        if not self.account_id:
             raise AWSError("Parameter 'mpa_account_id' must be provided to assume the role.")
```

Ruff (0.14.13)

Avoid specifying long messages outside the exception class

(TRY003)